library/objective-c/EnvoyEngineImpl.m

```diff
@@ -16,15 +16,21 @@ - (instancetype)init {
   return self;
 }
 
-- (int)runWithConfig:(NSString *)config {
-  return [self runWithConfig:config logLevel:@"info"];
+- (int)runWithConfig:(EnvoyConfiguration *)config logLevel:(NSString *)logLevel {
+  NSString *templateYAML = [[NSString alloc] initWithUTF8String:config_template];
+  NSString *resolvedYAML = [config resolveTemplate:templateYAML];
+  if (resolvedYAML == nil) {
+    return 1;
```

Comment:
This is the "throwing" when the yaml resolution fails?

Reply:
More or less yeah